ia-menu-slider.js

```diff
@@ -0,0 +1,3 @@
+import { IAMenuSlider } from './src/ia-menu-slider.js';
+
+window.customElements.define('ia-menu-slider', IAMenuSlider);
```

Comment:
Yeah, I think the idea is that if you just want to quickly grab the custom element definition, you grab the named file. Otherwise, you can import the class from index.js. I like the idea of the either/or for situations where you need to extend the class before defining the custom element.

Reply:
is this something we'd like to do for future repos as well - as part of our template?